packages/react-data-grid/src/common/editors/EditorPortal.tsx

```diff
@@ -1,43 +1,24 @@
-import React from 'react';
+import React, { useState, useLayoutEffect } from 'react';
 import ReactDOM from 'react-dom';
 
 interface Props {
   children: React.ReactNode;
-  target: Node;
+  target: Element;
```

Comment:
This better matches the type `createPortal` accepts
![image](https://user-images.githubusercontent.com/567105/57093885-ee7e5980-6d06-11e9-8e33-f5d8494b5cae.png)